x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/MachineLearning.java

```diff
@@ -656,6 +656,20 @@ public void loadExtensions(ExtensionLoader loader) {
         Property.NodeScope
     );
 
+    // Serverless only - trigger scale up if necessary
```

Comment:
```suggestion
    // The next two settings currently only have an effect in serverless. They can be set as overrides to
    // trigger a scale up of the ML tier so that it could accommodate the dummy entity in addition to the
    // whatever the standard autoscaling formula thinks is necessary.
```